### DISCUSSION TITLE
Version 1.14.2 is not showing up as an update for MacOS

### DISCUSSION THREAD

**Discussion by Stargator** [rancher-sandbox/rancher-desktop#7267] [Q&A] [answered]
I'm on MacOS 14.5 and have had Rancher Desktop for over a year.

I see that version 1.14.2 was released last month, but I've never been prompted to install it by the app. It doesn't even show it as an option:

<img width="1197" alt="Screenshot 2024-07-28 at 10 46 03 AM" src="https://github.com/user-attachments/assets/8eff5e53-2d78-42cd-930e-9477a936bead">

I've been able to update the app with previous updates. I've tried restarting my laptop, but it doe snot show an update is available.

Is there anything I can do to force it to re-check for updates?

1.14.2 has a change related to macOS and I'd like to get it.

**Comment by jandubois** [comment, marked as answer]
You'll have to install it manually; we mention this in the release notes:

> We are not enabling auto-update for the 1.14.2 release because the percentage of affected users will be very small. You need to manually update if you are on macOS and want to run `spin up`.

Manually installing is just downloading the `*.dmg` for your platform from the [1.14.2 release page](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.14.2) and then dragging Rancher Desktop into the Applications folder.

Alternatively you can wait for 1.15.0, which should be released in early August.

**Reply by Stargator** [reply]
Okay, I didn't dig into the notes. Thanks.
